packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.Documentation.cs

```diff
@@ -12,12 +14,14 @@ namespace Microsoft.Generator.CSharp
 {
     internal sealed partial class CodeWriter
     {
-        public CodeWriter WriteXmlDocumentationSummary(FormattableString? text)
+        private const string SingleArgFormat = "{0}";
+
+        public CodeWriter WriteXmlDocumentationSummary(IReadOnlyList<FormattableString>? text)
         {
             return WriteXmlDocumentation("summary", text);
         }
 
-        public CodeWriter WriteXmlDocumentation(string tag, FormattableString? text)
+        public CodeWriter WriteXmlDocumentation(string tag, IReadOnlyList<FormattableString>? text)
```

Comment:
I think empty collection and null collection here have the same meaning: we do not have a content here. Therefore I think we could make this non-nullable.
Also reading the other part of the changes, I did not see a place that we would construct a null value for a description yet